README.md

```diff
@@ -118,6 +118,27 @@ Then to verify the pipeline definition using the defined policies:
 
     make check-pipeline
 
+### Running policies against local [ec-cli] build
+
+Build a local version of `ec-cli` in your local ec-cli repo:
+
+    make build
+
+Create a `policy.yaml` file in your local `ec-cli` repo with something like:
+
+    ---
+    sources:
+      - policy:
+        - github.com/enterprise-contract/ec-policies//policy//lib
+        - github.com/enterprise-contract/ec-policies//policy//release
+      data:
+        - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
+        - github.com/release-engineering/rhtap-ec-policy//data
```

Comment:
Would we use local policies here, i.e from the filesystem?
```suggestion
        - <path-to>/ec-policies//policy//lib
        - <path-to>/ec-policies//policy//release
      data:
        - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
        - github.com/release-engineering/rhtap-ec-policy//data
```

Reply:
Done